code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -17,7 +17,8 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food)
 	var/doants = TRUE 							//! Will ants spawn to eat this food if it's on the floor
 	var/tmp/made_ants = FALSE 					//! Has this food already spawned ants
 	var/ant_amnt = 5 							//! How many ants are added to food / how much reagents removed?
-	var/sliceable = FALSE 						//! Can this food be sliced with a knife
+	var/sliceable = FALSE						//! Can this food be sliced
+	var/slice_tools = TOOL_CUTTING | TOOL_SAWING	//! Which tools can be used to slice this food
```

Comment:
Could we just default this to?
`slice_tools = TOOL_SNIPPING | TOOL_CUTTING | TOOL_SAWING`

 I don't see why only pizza gets the better cutting. Most things that can be cut by a knife can be cut by a scissor.